mmv1/third_party/terraform/services/bigquery/resource_bigquery_table.go

```diff
@@ -1405,9 +1412,12 @@ func resourceBigQueryTableRead(d *schema.ResourceData, meta interface{}) error {
 	if err := d.Set("max_staleness", res.MaxStaleness); err != nil {
 		return fmt.Errorf("Error setting max_staleness: %s", err)
 	}
-	if err := d.Set("labels", tpgresource.FlattenLabels(res.Labels, d)); err != nil {
+	if err := d.Set("labels", tpgresource.FlattenLabels(res.Labels, d, "labels")); err != nil {
```

Comment:
We may want to wrap the whole `d.Set` call into a function in order to assert that the target field and the `lineage` values match. It's a pretty easy copy-paste error to let those diverge at the moment.

Reply:
Done. A new func SetLabels is called here.